### PR TITLE
Update procedures descriptions and modes according to the test results

### PR DESCRIPTION
**Pull request by NataliaIvakina**
Update descriptions and modes of the procedures according to the [test results](https://trello.com/c/bCR3TJOn)

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1155.surge.sh

**Comment by renetapopova** [comment]
I started reviewing the PR, and I noticed that some of the descriptions in the docs had been improved, so we might consider improving the descriptions in the code base instead. Also, some of the mismatches are related to roles and labels. We should decide if we want to uncomment them or skip the slashes in the test. I'll leave it for now, and we can look into it together.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 1]
```diff
@@ -243,7 +243,7 @@ In 4.2, signature changed to `db.listLocks() :: (mode :: STRING, resourceType ::
 | xref:reference/procedures.adoc#procedure_dbms_checkconfigvalue[`dbms.checkConfigValue()`]
 | label:no[]
 | label:yes[]
-| New in 5.0.
+| label:admin-only[] New in 5.0.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:246]:
```suggestion
| label:new[Introduced in 5.0]  label:admin-only[]
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 2]
```diff
@@ -965,18 +957,8 @@ m|READ
...
-
-Valid _key: value_ pairs for the `options` map are:
-
-* `skip: <number>` -- skip the top N results.
-* `limit: <number>` -- limit the number of results returned.
-* `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.
-
-The `options` map and any of the keys are optional.
-An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
+Query the given full-text index. Returns the matching nodes, and their Lucene query score, ordered by score.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:960]: Let's see if we can update the code base instead. You can remove it for now.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 3]
```diff
@@ -992,18 +974,8 @@ m|READ
...
-
-Valid _key: value_ pairs for the `options` map are:
-
-* `skip: <number>` -- skip the top N results.
-* `limit: <number>` -- limit the number of results returned.
-* `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.
-
-The `options` map and any of the keys are optional.
-An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
+Query the given full-text index. Returns the matching relationships, and their Lucene query score, ordered by score.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:977]: the same here

File: modules/ROOT/pages/reference/procedures.adoc [hunk 4]
```diff
@@ -1018,26 +990,13 @@ m|READ
...
-+
-image::euclidean_similarity_equation.svg["The Euclidean of vector v and vector u is defined as 1 over the quantity 1 plus the square of the l2-norm of vector v subtract vector u, which exists in the bounded set of real numbers between 0 exclusive and 1 inclusive."]
-+
-
-* `"cosine"` -- Cosine similarity, where:
-+
-image::cosine_similarity_equation.svg["The cosine of vector v and vector u is defined as half of the quanity 1 plus the scalar product of v hat u hat, which equals half of the quantity 1 plus the scalar product of vector v vector u over the product of the l2-norm of vector v and the l2 norm ov vector u, which exists in the bounded set of real numbers between 0 inclusive and 1 inclusive."]
-In the above equation, the trigonometric cosine is given by the scalar product of the two unit vectors.
-+
+Create a named node vector index for the given label and property for a specified vector dimensionality.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:993]: same here

File: modules/ROOT/pages/reference/procedures.adoc [hunk 5]
```diff
@@ -1602,13 +1556,11 @@ CALL dbms.cluster.readReplicaToggle("neo4j", false)
 |===
 | Description
 a|
-The toggle can pause or resume the pulling of new transactions for a specific database.
-If paused, the database secondary does not pull new transactions from the other cluster members for the specific database.
-The database secondary is still available for reads, you can perform a backup, etc.
+The toggle can pause or resume the secondary replication process
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:1559]:
```suggestion
The toggle can pause or resume the secondary replication process.
```
And update the code base with the period.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 6]
```diff
@@ -1748,7 +1700,7 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/{page-ver
 |===
 | Description
 a|
-List capabilities.
+List capabilities
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:1703]:
```suggestion
List capabilities.
```
Update the code as well.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 7]
```diff
@@ -1761,19 +1713,7 @@ m|DBMS
...
-Results include:
-
-* `value`: the current value of the setting.
-* `dynamic`: true if the value can be changed at runtime, or false if it requires a restart.
-* `defaultValue`: the default value of the setting.
-* `startupValue`: the value when the server was started, after _neo4j.conf_ and command line arguments have been applied.
-* `explicitlySet`: true if the value was set explicitly, or false if it was set by default.
-* `validValues`: contains information on data types and possible values for the settings.
-
+List the currently active config of Neo4j.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:1716]:
```suggestion
List the currently active configuration settings of Neo4j.
```
Also update the code base.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 8]
```diff
@@ -1825,8 +1765,7 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/{page-ver
 |===
 | Description
 a|
-Place a database in quarantine or remove thereof.
-It must be executed over `bolt://`.
+Place a database into quarantine or remove from it.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:1768]:
```suggestion
Place a database into quarantine or remove it from it.
```
Fix it in the code base as well.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 9]
```diff
@@ -1841,10 +1780,7 @@ m|DBMS
 |===
 | Description
 a|
-Query JMX management data by domain and name.
-
-Valid queries should use the syntax outlined in the link:https://docs.oracle.com/en/java/javase/11/docs/api/java.management/javax/management/ObjectName.html[javax.management.ObjectName API documentation]. +
-For instance, use `+"*:*"+` to find all JMX beans.
+Query JMX management data by domain and name. For instance, ":"
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:1783]:
```suggestion
Query JMX management data by domain and name. For instance, use `:` to find all JMX beans.
```
Also in the code.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 10]
```diff
@@ -1874,7 +1811,8 @@ m|DBMS
 [cols="<15s,<85"]
 |===
 | Description
-a| With this method you can set the allocator, which is responsible to select servers for hosting databases. The only current option is `EQUAL_NUMBERS`.
+a|
+With this method you can set the allocator, which is responsible to select servers for hosting databases.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:1815]:
```suggestion
With this method, you can set the allocator that is responsible for selecting servers for hosting databases.
```
also in the code

File: modules/ROOT/pages/reference/procedures.adoc [hunk 11]
```diff
@@ -1994,10 +1932,9 @@ m|DBMS
 |===
 | Description
 a|
-Update a given setting value.
-Passing an empty value results in removing the configured value and falling back to the default value.
-Changes do not persist and are lost if the server is restarted.
-In a clustered environment, `dbms.setConfigValue` affects only the cluster member it is run against.
+Updates a given setting value.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:1935]: Update in the code instead.

**Review by renetapopova — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/reference/procedures.adoc (hunk 2 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:960]:
```suggestion
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 12]
```diff
@@ -965,18 +957,8 @@ m|READ
...
-Valid _key: value_ pairs for the `options` map are:
-
-* `skip: <number>` -- skip the top N results.
-* `limit: <number>` -- limit the number of results returned.
-* `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.
-
-The `options` map and any of the keys are optional.
-An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
+Query the given full-text index. Returns the matching nodes, and their Lucene query score, ordered by score.
+Valid keys for the options map are: 'skip' to skip the top N results; 'limit' to limit the number of results returned; 'analyzer' to use the specified analyzer as search analyzer for this query.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:961]:
```suggestion
```

File: modules/ROOT/pages/reference/procedures.adoc (hunk 3 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:977]:
```suggestion
Query the given full-text index.

Returns the matching relationships and their Lucene query score, ordered by score.

Valid _key: value_ pairs for the `options` map are:

* `skip: <number>` -- skip the top N results.
* `limit: <number>` -- limit the number of results returned.
* `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.

The `options` map and any of the keys are optional.
An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 13]
```diff
@@ -992,18 +974,8 @@ m|READ
...
-Valid _key: value_ pairs for the `options` map are:
-
-* `skip: <number>` -- skip the top N results.
-* `limit: <number>` -- limit the number of results returned.
-* `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.
-
-The `options` map and any of the keys are optional.
-An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
+Query the given full-text index. Returns the matching relationships, and their Lucene query score, ordered by score.
+Valid keys for the options map are: 'skip' to skip the top N results; 'limit' to limit the number of results returned; 'analyzer' to use the specified analyzer as search analyzer for this query.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:978]:
```suggestion
```

File: modules/ROOT/pages/reference/procedures.adoc (hunk 4 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:993]:
```suggestion
Create a named vector index for the given label, and property with given vector dimensionality, using the given similarity function.

`vectorDimension` must be between `1` and `2048` _inclusively_.

Valid case-insensitive values for the `vectorSimilarityFunction` are:

* `"euclidean"` -- Euclidean similarity, where:
+
image::euclidean_similarity_equation.svg["The Euclidean of vector v and vector u is defined as 1 over the quantity 1 plus the square of the l2-norm of vector v subtract vector u, which exists in the bounded set of real numbers between 0 exclusive and 1 inclusive."]
+

* `"cosine"` -- Cosine similarity, where:
+
image::cosine_similarity_equation.svg["The cosine of vector v and vector u is defined as half of the quanity 1 plus the scalar product of v hat u hat, which equals half of the quantity 1 plus the scalar product of vector v vector u over the product of the l2-norm of vector v and the l2 norm ov vector u, which exists in the bounded set of real numbers between 0 inclusive and 1 inclusive."]
In the above equation, the trigonometric cosine is given by the scalar product of the two unit vectors.
+
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 14]
```diff
@@ -1018,26 +990,13 @@ m|READ
...
-image::euclidean_similarity_equation.svg["The Euclidean of vector v and vector u is defined as 1 over the quantity 1 plus the square of the l2-norm of vector v subtract vector u, which exists in the bounded set of real numbers between 0 exclusive and 1 inclusive."]
-+
-
-* `"cosine"` -- Cosine similarity, where:
-+
-image::cosine_similarity_equation.svg["The cosine of vector v and vector u is defined as half of the quanity 1 plus the scalar product of v hat u hat, which equals half of the quantity 1 plus the scalar product of vector v vector u over the product of the l2-norm of vector v and the l2 norm ov vector u, which exists in the bounded set of real numbers between 0 inclusive and 1 inclusive."]
-In the above equation, the trigonometric cosine is given by the scalar product of the two unit vectors.
-+
+Create a named node vector index for the given label and property for a specified vector dimensionality.
+Valid similarity functions are 'EUCLIDEAN' and 'COSINE', and are case-insensitive.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:994]:
```suggestion
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 15]
```diff
@@ -1018,26 +990,13 @@ m|READ
...
+Use the `db.index.vector.queryNodes` procedure to query the named index.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:995]:
```suggestion
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 16]
```diff
@@ -957,8 +957,18 @@ m|READ
...
+Returns the matching nodes and their Lucene query score, ordered by score.
+
+Valid _key: value_ pairs for the `options` map are:
+
+* `skip: <number>` -- skip the top N results.
+* `limit: <number>` -- limit the number of results returned.
+* `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.
+
+The `options` map and any of the keys are optional.
+An example of the `option
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:971]: This sentence is missing its end.

File: modules/ROOT/pages/reference/procedures.adoc [hunk 17]
```diff
@@ -1518,8 +1509,8 @@ m|WRITE
 |===
 | Description
 a|
-Returns endpoints of this instance.
-Used in disaster recovery.
+Returns the advertised bolt capable endpoints for a given database, divided by each endpoint’s capabilities.
+For example an endpoint may serve read queries, write queries and/or future `getRoutingTable` requests.
```
Comment (renetapopova) [inline modules/ROOT/pages/reference/procedures.adoc:1513]:
```suggestion
For example, an endpoint may serve read queries, write queries, and/or future `getRoutingTable` requests.
```

File: modules/ROOT/pages/reference/procedures.adoc (hunk 11 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:1968]:
```suggestion
Update a given setting value.
Passing an empty value results in removing the configured value and falling back to the default value.
Changes do not persist and are lost if the server is restarted.
In a clustered environment, `dbms.setConfigValue` affects only the cluster member it is run against.
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 18]
```diff
@@ -1994,10 +1932,9 @@ m|DBMS
 |===
 | Description
 a|
-Update a given setting value.
-Passing an empty value results in removing the configured value and falling back to the default value.
-Changes do not persist and are lost if the server is restarted.
-In a clustered environment, `dbms.setConfigValue` affects only the cluster member it is run against.
+Updates a given setting value.
+Passing an empty value will result in removing the configured value and falling back to the default value.
+Changes will not persist and will be lost if the server is restarted.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:1970]:
```suggestion
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 19]
```diff
@@ -1994,10 +1932,9 @@ m|DBMS
 |===
 | Description
 a|
-Update a given setting value.
-Passing an empty value results in removing the configured value and falling back to the default value.
-Changes do not persist and are lost if the server is restarted.
-In a clustered environment, `dbms.setConfigValue` affects only the cluster member it is run against.
+Updates a given setting value.
+Passing an empty value will result in removing the configured value and falling back to the default value.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:1969]:
```suggestion
```

File: modules/ROOT/pages/reference/procedures.adoc [hunk 20]
```diff
@@ -976,7 +968,7 @@ Valid _key: value_ pairs for the `options` map are:
 * `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.
 
 The `options` map and any of the keys are optional.
-An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
+An example of the `option
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/reference/procedures.adoc:971]:
```suggestion
An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
```

**Pull request merged**